.github/workflows/nix.yaml

```
@@ -14,21 +19,43 @@ permissions:
+        uses: actions/checkout@v4
       
       - name: Apply labels
-        uses: actions/labeler@v4
         if: github.event_name == 'pull_request' && github.event.action != 'closed'
+        uses: actions/labeler@v4
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
       
+      - name: Check conditions
+        id: evaluate
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:31: Double quote to prevent globbing and word splitting

(shellcheck)